server/src/modules/content-pages/controller.ts

```diff
@@ -24,7 +24,9 @@ export default class ContentPageController {
 	public static async getContentPage(
 		idOrPath: number | string,
 		user: Avo.User.User | null,
-		request: Request | null
+		request: Request | null,
+		shouldResolveMedia = true // Set this to true by default, so we don't change the behavior of other code that already
+		// uses this function. We only want to pass false in the /duplicate route
```

Comment:
These comments were explanation for you

We might want to make this more clear for the developer in the code:
```suggestion
		shouldResolveMedia = true // Will resolve media tile block to include their tiles and media players to include a play token
```